app/components/schedule/Custom.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="dayblock">
+    <div class="container mx-auto sm:flex show-basic-infos">
+      <div class="mr-4 timing-infos">
+        <div class="mb-2 time-block sm:mb-0">
+          {{ removeSeconds(show.start) }}
+          <img src="@/assets/img/arrow-schedule.svg" alt="" class="inline-block w-8 pb-1">
+          {{ removeSeconds(show.end) }}
+        </div>
+      </div>
+      <div class="flex justify-between w-full schedule-infos">
+        <div class="onair-infos">
+          <div v-if="show.archive_lahmastore_base_url.includes(currentHost)">
+            <NuxtLink :to="show.archive_lahmastore_base_url.replace(currentHost, '')">
+              <b>{{ show.name }}</b>
+            </NuxtLink>
+          </div>
+          <div v-else-if="show.archive_lahmastore_base_url.includes('lahmacun.hu/shows/')">
+            <NuxtLink :to="'/shows/' + show.archive_lahmastore_base_url.substring(show.archive_lahmastore_base_url.lastIndexOf('/') + 1)">
```

Comment:
@baxgas this doesn't seem to work for the setting I added now in WP (which should be the actual setting for the upcoming custom event). The URL doesn't see to include the last part, I only have `localhost/shows` link in my local test. Can you please check?